features/steps/steps.py

```diff
@@ -36,11 +29,38 @@ def when_i_run_command(context, command, user_spec):
     context.process = process
 
 
+@when("I attach `{contrac_token}` {user_spec}")
+def when_i_attach_token(context, contract_token, user_spec):
+    token = context.config.contract_token
+    if not token:
+        context.feature.skip(
```

Comment:
I have an idea here using before_scenario, I'll post that once I have a working suggestion

Reply:
show-skipped = True is default. You can add to cli `--no-skipped` and then do not print skipped step/feature/scenario, but still, it doesn't show: 
`context.feature.skip("Skipping test because you didn't provide a token")`